savegame.json

```diff
@@ -224,6 +224,14 @@
       {
         "displayed": "Requires Isolate",
         "normalized": "requires_isolate"
+      },
+      {
+        "displayed": "Phases",
```

Comment:
this tag doesn't seem used in the review

Reply:
It's the tag for the pre-existing phase change hook (as documented in `CUSTOM.md`). That hook already exists, but no extant objects in the mod use it, so the tag never made into `savegame.json`. It's not technically part of this feature, so perhaps I should have made it a separate PR, but I believe it's worth putting in the mod for the benefit of anyone who wants to use the phase change hook.

Reply:
ah apparently i completely forgot about me adding that feature, all good